mlflow/environment_variables.py

```diff
@@ -752,3 +752,8 @@ def get(self):
 MLFLOW_ASYNC_TRACE_LOGGING_RETRY_TIMEOUT = _EnvironmentVariable(
     "MLFLOW_ASYNC_TRACE_LOGGING_RETRY_TIMEOUT", int, 60
 )
+
+
+#: Specified the ID of the LoggedModel to link traces to.
+#: (default: ``None``)
+MLFLOW_ACTIVE_MODEL_ID = _EnvironmentVariable("MLFLOW_ACTIVE_MODEL_ID", str, None)
```

Comment:
Add env var so that we can track active model id inside subprocesses or setting it in serving envs